site/src/hooks/useMe.ts

```diff
@@ -1,14 +1,12 @@
 import { User } from "api/typesGenerated";
 import { useAuth } from "components/AuthProvider/AuthProvider";
-import { isAuthenticated } from "xServices/auth/authXService";
 
 export const useMe = (): User => {
-  const [authState] = useAuth();
-  const { data } = authState.context;
+  const { user } = useAuth();
```

Comment:
Connected to one of my previous questions: I'm wondering if `useMe` has as much value now, since it's just using the new `useAuth` under the hood?

Reply:
Or maybe it's still useful, since these are basically "pre-configured", and you don't have to think about where to get the value from